contracts/interchain-token-service/src/state.rs

```diff
@@ -60,24 +78,25 @@ pub enum MessageDirection {
 
 /// The deployment type of the token.
 pub enum TokenDeploymentType {
-    /// The token is trustless, i.e only owned by ITS.
+    /// The token is trustless, i.e. only owned by ITS.
     Trustless,
     /// The token has a custom minter.
     CustomMinter,
 }
 
 /// Global config for a token.
 #[cw_serde]
-pub struct TokenConfig {
+pub struct GlobalTokenConfig {
```

Comment:
Weird to explicitly label it as Global, I'd prefer renaming the other struct to make it more obvious that it's per chain. Similar to all verifiers in service registry contract vs verifiers registered by chain.

Reply:
removed "global" again